components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/EventEntity.java

```diff
@@ -104,6 +104,10 @@
     @CollectionTable(name = "ord_labels", joinColumns = @JoinColumn(name = "event_definition_id"))
     private List<Label> labels;
 
+    @ElementCollection
+    @CollectionTable(name = "event_api_definition_extensible", joinColumns = @JoinColumn(name = "event_definition_id"))
+    private List<ExtensibleEntity> extensibleEntity;
```

Comment:
For consistency reasons with the spec, this field must be renamed to `extensible`